src/main/scala/ca/ferlab/ferload/client/commands/Configure.scala

```diff
@@ -0,0 +1,86 @@
+package ca.ferlab.ferload.client.commands
+
+import ca.ferlab.ferload.client.clients.inf.{ICommandLine, IFerload}
+import ca.ferlab.ferload.client.commands.factory.{BaseCommand, CommandBlock}
+import ca.ferlab.ferload.client.configurations._
+import com.typesafe.config.Config
+import org.apache.commons.lang3.StringUtils
+import org.json.JSONObject
+import picocli.CommandLine.{Command, Option}
+
+import java.util.Optional
+
+@Command(name = "configure", mixinStandardHelpOptions = true, description = Array(" Help configure this tools."), version = Array("0.1"))
+class Configure(userConfig: UserConfig, appConfig: Config, commandLine: ICommandLine, ferload: IFerload) extends BaseCommand(appConfig) with Runnable {
+
+  @Option(names = Array("-f", "--ferload-url"), description = Array("Ferload url"))
+  var ferloadUrl: Optional[String] = Optional.empty
```

Comment:
picocli only support java Optional not scala Option

Reply:
Not sure but maybe there is an implicit conversion in scala

Reply:
Like [this](https://www.scala-lang.org/api/2.13.5/scala/jdk/OptionConverters$.html)

Reply:
Perfect will try this thx